provisioning/roles/mailcatcher/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+- name: ensure needed packages are installed
+  apt: pkg={{ item }} state=latest
+  sudo: yes
+  with_items:
+    - libsqlite3-dev
+
+- name: ensure mailcatcher installed
+  gem: name=mailcatcher user_install=no state=latest
+  sudo: yes
```

Comment:
I just updated the solr role along with the supervisord one.

You now have multiple handlers for supervisord to restart it completly or just some services.